README.md

```diff
@@ -81,6 +81,10 @@ As an example:
 - `open` is true when the auto-complete result list is visible
 - `value` is the selected value from the list or the empty string when cleared
 
+## Properties
+
+- `fetchResult` is the method that will be call to fetch the results
```

Comment:
I like this! It might be good to provide an example here even
```suggestion
- `fetchResult` you can override the default method used to query for results by overriding this property: `document.querySelector('auto-complete').fetchResult = async (el, url) => (await fetch(url)).text()`
```